lib/gds_api/content_api.rb

```diff
@@ -52,13 +52,12 @@ def child_tags(tag_type, parent_tag, options={})
   end
 
   def tag(tag, tag_type=nil)
-    url = "#{base_url}/tags"
-
-    if tag_type
-      url << "/#{CGI.escape(tag_type)}"
+    if tag_type.nil?
```

Comment:
I do wonder if it's worth just defaulting the type to `"section"`, as that's the current behaviour.